_modals/modal-modal-with-alert.php

```diff
@@ -0,0 +1,26 @@
+<div id="modalWithAlert" class="modal" tabindex="-1" role="dialog">
+  <div class="modal-dialog modal-sm">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+            aria-hidden="true">&times;</span></button>
+        <h3 class="modal-title text-center">Activate Your Account</h3>
+      </div>
+      <div class="modal-body">
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-xs-12">
+              <div class="alert alert-danger" role="alert">Payment method has been successfully deleted.</div>
```

Comment:
@Omicron7 you can use this patter here. The alert markup goes into the modal body right after the first .container-fluid > .row > .col-xs-12